### PR TITLE
[INVE-11559] Polyline rendering - docs update

### PR DESCRIPTION
**Pull request by jaroslawjanas**
This is a PR for updating polyline map rendering docs.

### CODE REVIEW

**Comment by jaroslawjanas** [comment]
I think there is no need to single out lines for popups, since they work with all shapes.
`FYI: the popups we mentioned work` 🎉 

This link `<<Stored Layer Configurations, when in Visualization edit mode>>` is broken and I'm not exactly sure which section it should be pointing to.
@corrigancd

File: docs/modules/siren-investigate/pages/enhanced_tilemap.adoc [hunk 1]
```diff
@@ -588,7 +590,7 @@ NOTE: Layers will be automatically positioned at the top or bottom of their type
 
 * Markers
```
Comment (corrigancd) [inline docs/modules/siren-investigate/pages/enhanced_tilemap.adoc:591]: please remove `Markers`. These have been removed

**Comment by corrigancd** [comment]
> I think there is no need to single out lines for popups, since they work with all shapes.
> `FYI: the popups we mentioned work` 
> 
> This link `<<Stored Layer Configurations, when in Visualization edit mode>>` is broken and I'm not exactly sure which section it should be pointing to.
> @corrigancd

I honestly can't find where the broken link is?

**Comment by jaroslawjanas** [comment]
Fixed the link in https://github.com/sirensolutions/enhanced_tilemap/pull/271/commits/0933fcf07b019be73678b9f013fc695850269cdb

**Pull request merged**